### PR TITLE
react with redux simple app

### PR DESCRIPTION
**Pull request by komarnytskyy**

### CODE REVIEW

File: src/reducers/index.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    return [
+        {title: 'No Scrubs', duration: '4:05'},
+        {title: 'Macarena', duration: '2:30'},
+        {title: 'All Star', duration: '3:15'},
+        {title: 'I Want it That Way', duration: '1:45'},
+    ]
+}
+
+const selectedSongReducer = (selectedSong = null, action) => {
+    if (action.type === 'SONG_SELECTED') {
```
Comment (oleksandr-rokyta-by) [inline src/reducers/index.js]: It's better to use here constant instead of 'SONG_SELECTED'

File: src/components/SongList.tsx [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+      </div>
+    )
+  }
+}
+
+const getState = (state: any) => {
+  return { songs: state.songs };
+}
+
+export default connect(getState, { selectSong: selectSong })(SongList);
```
Comment (oleksandr-rokyta-by) [inline src/components/SongList.tsx]: You can just set {selectSong}

**Review by oleksandr-rokyta-by — CHANGES_REQUESTED** [review]

File: src/reducers/index.js (hunk 1 above)
Comment (komarnytskyy) [reply src/reducers/index.js]: fixed

File: src/components/SongList.tsx (hunk 2 above)
Comment (komarnytskyy) [reply src/components/SongList.tsx]: fixed

**Review by oleksandr-rokyta-by — APPROVED** [review]

**Pull request merged**